package echo;

option cc_generic_services = true;

message EchoRequest {
  required string str = 1;
}

message EchoResponse {
  required string str = 1;
}

service EchoService {
  rpc Echo (EchoRequest) returns (EchoResponse);
}

